ivy/functional/frontends/torch/miscellaneous_ops.py

```diff
@@ -5,19 +5,9 @@ def flip(input, dims):
     return ivy.flip(input, dims)
 
 
-flip.unsupported_dtypes = (
-    "uint16",
-    "uint32",
-    "uint64",
-)
-
-
 def roll(input, shifts, dims=None):
     return ivy.roll(input, shifts, dims)
 
 
-roll.unsupported_dtypes = (
-    "uint16",
-    "uint32",
-    "uint64",
-)
+def repeat_interleave(input, repeats, dim=None):

```

Comment:
The torch `repeat_interleave` function should also take in an optional keyword argument `output_size`, you can see it [here](https://pytorch.org/docs/stable/generated/torch.repeat_interleave.html)

Reply:
added